src/model/scripts/pytorch_lighting_training/pl_configurations/dataset/default_dataset.yaml

```diff
@@ -0,0 +1,4 @@
+training_dataset: /path/to/train/data
```

Comment:
Should we also do `example.yaml` thing here?

Reply:
Done